test/commands/generate/models.test.ts

```diff
@@ -7,6 +7,16 @@ const generalOptions = ['generate:models'];
 const outputDir = './test/commands/generate/models';
 
 describe('models', () => {
+  test
+    .stderr()
+    .stdout()
+    .command([...generalOptions, 'typescript', 'http://bit.ly/asyncapi'])
```

Comment:
oh, another test that can kick us in our buts in the future, but yeah we need to solve https://github.com/asyncapi/cli/issues/161 first